tests/Unit/SDK/Common/Attribute/AttributesTest.php

```diff
@@ -41,6 +41,16 @@ public function test_numeric_attribute_name(): void
         }
     }
 
+    public function test_integer_attribute_key_is_cast_to_string(): void
```

Comment:
Test above (`test_numeric_attribute_name`) can be removed, duplicates this test.